ui/src/types/tasks.ts

```diff
@@ -1,6 +1,36 @@
 import {Task as ITask} from 'src/client'
-import {Label} from 'src/types'
+import {Label, NormalizedState, Run, RemoteDataState, LogEvent} from 'src/types'
 
 export interface Task extends ITask {
   labels?: Label[]
```

Comment:
I feel like this can be required since we are adding defaults?

Reply:
Good catch!  I wanted to do the bare minimum until I normalize the `Labels` resource.  All of this is going to change when labels within a resource are just a list of label ids.